app/routes/agenda-items/session.ts

```diff
@@ -15,14 +15,14 @@ export default class AgendaItemsAgendaItemSessionRoute extends Route {
 
     const session = await this.store.findRecord(
       'session',
-      sessions?.firstObject?.id as string,
+      sessions?.objectAt(0)?.id as string,
```

Comment:
`.objectAt` is an EmberArray thing which doesn't exist here anymore (I think EmberData v5 removed that, but it should have shown a deprecation in v4 as well).

```suggestion
      sessions?.at(0)?.id as string,
```